uberfire-rest/uberfire-rest-backend/src/main/java/org/guvnor/rest/backend/JobRequestHelper.java

```diff
@@ -490,170 +478,67 @@ public JobResult createOrganizationalUnit(final String jobId,
             }
         }
 
-        OrganizationalUnit organizationalUnit = organizationalUnitService.getOrganizationalUnit(organizationalUnitName);
+        OrganizationalUnit organizationalUnit = organizationalUnitService.getOrganizationalUnit(spaceName);
         if (organizationalUnit != null) {
             result.setStatus(JobStatus.BAD_REQUEST);
-            result.setResult("OrganizationalUnit with name " + organizationalUnitName + " already exists");
+            result.setResult("OrganizationalUnit with name " + spaceName + " already exists");
```

Comment:
Same as my previous comment. Maybe it would be better to use "Space" in the result.